docs/world api.md

```diff
@@ -85,7 +85,7 @@ inside a World object.
 ### Player Options
 
 Players provide customized settings for their World in the form of yamls.
-Those are accessible through `self.world.<option_name>[self.player]`. A dict
+Those are accessible through `self.world.options["<option_name>"]`. A dict
```

Comment:
The documentation being replaced specifies the player. Shouldn't the replacement also specify the player?